CVE-2022-25237/data/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+
+services:
+  bonita:
+    image: bonita:7.13.0
```

Comment:
Please download the Docker image and keep it in the data folder. Install it from the file.